pom.xml

```diff
@@ -218,35 +218,6 @@
         </executions>
       </plugin>
 
-      <plugin>
```

Comment:
What was the motivation for removing this?

Reply:
It was failing the build on Java 8 lambda constructs it didn't understand.

Reply:
I can't tell if we're running on different environments, but I only get these checkstyle warnings:
```
[WARNING] src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java:[298] (javadoc) NonEmptyAtclauseDescription: At-clause should have a non-empty description.
[WARNING] src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java:[344] (javadoc) NonEmptyAtclauseDescription: At-clause should have a non-empty description.
```

Reply:
Maybe something works differently on Travis. The following log is from the very first commit: 
4c2dee5

https://travis-ci.org/GoogleCloudPlatform/appengine-plugins-core/builds/351041235?utm_source=github_status&utm_medium=notification

```
[INFO] Building jar: /home/travis/build/GoogleCloudPlatform/appengine-plugins-core/target/appengine-plugins-core-0.5.2-SNAPSHOT-sources.jar
[INFO] com already added, skipping
[INFO] com/google already added, skipping
[INFO] com/google/cloud already added, skipping
[INFO] com/google/cloud/tools already added, skipping
[INFO] com/google/cloud/tools/libraries already added, skipping
[INFO] 
[INFO] --- maven-checkstyle-plugin:2.17:check (google-checks) @ appengine-plugins-core ---
[WARNING] src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java:[298] (javadoc) NonEmptyAtclauseDescription: At-clause should have a non-empty description.
[WARNING] src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java:[344] (javadoc) NonEmptyAtclauseDescription: At-clause should have a non-empty description.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:47 min
[INFO] Finished at: 2018-03-08T21:50:22Z
[INFO] Final Memory: 41M/1556M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (google-checks) on project appengine-plugins-core: You have 2 Checkstyle violations. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```